### PR TITLE
Add Rails guide, starting with Sqeel guidelines

### PR DESCRIPTION
**Pull request by andyw8**

### CODE REVIEW

File: rails-guidelines/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Rails Guidelines
+
+* We use the [Squeel][squeel] gem, but it is no longer maintained.
+  We are still decidiing whether to remove it completely, or replace it
+  with [baby_sqeel][baby_sqeel] – see [Slack discussion][slack_squeel].
+  Regardless, we should avoid Squeel syntax when writing any _new_ code.
```
Comment (thisduck) [inline rails-guidelines/README.md]: Let's say we move to baby_squeel, why should we avoid that syntax? it makes some queries much easier to read and write.

Comment (andyw8) [reply rails-guidelines/README.md]: How about we postpone that discussion until after we move to baby_sqeel?

Comment (karenjho) [reply rails-guidelines/README.md]: I would like to have this discussion around specific queries which certain consider to be easier to read/write. Without the examples, I disagree that baby_squeel makes queries easier.

**Comment by andyw8** [comment]
Got a verbal OK from @thisduck to merge this.

**Pull request merged**